packages/plugin-relay/package.json

```diff
@@ -57,5 +57,8 @@
     "graphql-subscriptions": "^2.0.0",
     "graphql-tag": "^2.12.6"
   },
+  "engines": {
+    "node": ">=12.0.0"
```

Comment:
Added this so ESLint doesn't complain that `BigInt` doesn't exist yet, by default eslint checks down to 8.0.0 if no engines are provided.